### DISCUSSION TITLE
Use as formatter?

### DISCUSSION THREAD

**Discussion by mecampbellsoup** [cloudflare/pint#1113] [Q&A]
Hi guys! Cool project, thank you 🙏 

Just a super quick question: will this library format input PromQL queries?

**Comment by prymitive** [comment]
No, it's a linter/validator, not a formater.
Please use discussions for general questions.

**Reply by mecampbellsoup** [reply]
Thanks. Are you aware of any PromQL formatters out there?

**Reply by PrayagS** [reply]
@mecampbellsoup I've been using `promtool promql format` for quite a while now. Does the job.
